modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/ClientExtensionProjectConfigurator.java

```diff
@@ -1216,6 +1217,24 @@ else if (Objects.equals(clientExtension.type, "siteInitializer")) {
 				clientExtension, "membershipType", "open", "private",
 				"restricted");
 		}
+		else if (Objects.equals(clientExtension.type, "themeCSS")) {
+			_validateTypeSettingsValues(
+				clientExtension, "scope", "controlPanel", "layout");
+		}
+	}
+
+	private void _validateGlobalJSInstanceScope(
+		ClientExtension clientExtension) {
+
+		Map<String, Object> typeSettings = clientExtension.typeSettings;
+
+		String scope = String.valueOf(typeSettings.get("scope"));
+
+		if ((scope != null) && scope.equalsIgnoreCase("instance")) {
+			throw new GradleException(
+				"The value 'instance' for the 'scope' attribute was " +
+					"deprecated. Please use 'company'.");
```

Comment:
Failing the build is not "deprecated", so let's just say it's "no longer valid" instead. Also, I let's print the ClientExtension name so they can find it faster. I think we already do this in `_validateTypeSettingsValues`.

Reply:
Cool, will do. Thanks man